examples/rabbitmq-transaction-consumer/rabbitmq_transaction_consumer.out

```diff
@@ -3,10 +3,10 @@
 # Navigate to the directory that contains the
 # 'rabbitmq_transaction_consumer.bal' file, and run the 'ballerina run' command below.
 
-ballerina run --experimental rabbitmq_transaction_consumer.bal
+bal run rabbitmq_transaction_consumer.bal
```

Comment:
Missing `--experimental` flag?

Reply:
You don't need `--experimental` to run this.